web/src/stream_data.ts

```diff
@@ -575,7 +575,7 @@ export function can_unsubscribe_others(sub: StreamSubscription): boolean {
     );
 }
 
-export function can_post_messages_in_stream(stream: StreamSubscription): boolean {
+export let can_post_messages_in_stream = function (stream: StreamSubscription): boolean {
```

Comment:
Should this be part of previous commits?

Reply:
I have made this change while updating the test files. So, I have kept this change in this commit.

Reply:
Previous commits would fail without this change, so that means, it should be part of the previous commit.

Reply:
Moved the changes to first commit.